lib/runtime/newWasmer/memory.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2023 ChainSafe Systems (ON)
+// SPDX-License-Identifier: LGPL-3.0-only
+
+package newWasmer
```

Comment:
I didn't get the point to create a new package to place the wrapper, is that a problem in placing it in the `runtime/wasmer` package?

Reply:
It is just a temporary folder to me to build out wasmer with the newest version. If I try to use both versions in the same file, it doesn't compile, so I wrote out newWasmer with the updated version and then used it to replace our old wasmer.

Can see here that there is no more newWasmer because I made my rewrite the default https://github.com/ChainSafe/gossamer/pull/3168

Reply:
That's a good approach! I'm doing the same with the cli